server/src/main/java/com/objectcomputing/checkins/services/team/member/TeamMemberServicesImpl.java

```diff
@@ -108,4 +110,26 @@ public TeamMember update(@NotNull @Valid TeamMember teamMember) {
 
         return teamMembers;
     }
+
+    public void delete(@NotNull UUID id) {
+        MemberProfile currentUser = currentUserServices.getCurrentUser();
+        boolean isAdmin = currentUserServices.isAdmin();
+
+        TeamMember teamMember = teamMemberRepo.findById(id).orElse(null);
+        if(teamMember != null) {
+            Set<TeamMember> teamLeads = this.findByFields(teamMember.getTeamid(), null, true);
+
+            if (teamMemberRepo.findById(id).isEmpty()) {
+                throw new NotFoundException(String.format("Unable to locate teamMember to delete with id %s", id));
```

Comment:
Looks like this statement unreachable. Since `teamMemberRepo.findById(id).isEmpty()` is always `false`

Reply:
Agreed... @hwilliams314 could you remove the check above?

Reply:
done